src/hotspot/cpu/ppc/vm_version_ppc.cpp

```diff
@@ -306,9 +306,8 @@ void VM_Version::initialize() {
     FLAG_SET_DEFAULT(UseAESCTRIntrinsics, false);
   }
 
-  if (UseGHASHIntrinsics) {
-    warning("GHASH intrinsics are not available on this CPU");
-    FLAG_SET_DEFAULT(UseGHASHIntrinsics, false);
+  if (FLAG_IS_DEFAULT(UseGHASHIntrinsics) && VM_Version::has_vsx()) {
+    FLAG_SET_DEFAULT(UseGHASHIntrinsics, true);
```

Comment:
An attempt to enable `UseGHASHIntrinsics` should be rejected if `!VM_Version::has_vsx()` and a warning printed. See handling of other flags.
Alternatively, we could require Power8 by doing [JDK-8331859](https://bugs.openjdk.org/browse/JDK-8331859) first (or at least a part of it).